openshift/unit-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+echo "Running unit-tests.sh"
+
+# Ensure that some home var is set and that it's not the root
+export HOME=${HOME:=/tmp/kubebuilder/testing}
+if [ $HOME == "/" ]; then
+  export HOME=/tmp/kubebuilder/testing
+fi
+
+GOFLAGS='-mod=readonly' make go-test
```

Comment:
This should be pointing to the right units target in the root Makefile
```suggestion
GOFLAGS='-mod=readonly' make unit-test
```